benchmarks/nemo/nemolite2d/psykal/Makefile_gen

```diff
@@ -99,10 +99,10 @@ clean:
 
 # OpenCL Execution helpers
 run_opencl_jit:
-	 DL_ESM_ALIGNMENT=64 FORTCL_KERNELS_FILE=allkernels.cl ./nemolite2d.exe
+	 DL_ESM_ALIGNMENT=64 FORTCL_KERNELS_FILE=opencl_kernels_0.cl ./nemolite2d.exe
 
 run_xilinx_sw_emu:
-	FORTCL_KERNELS_FILE=allkernels.xclbin XCL_EMULATION_MODE=sw_emu ./nemolite2d.exe
+	FORTCL_KERNELS_FILE=device_binary.xclbin XCL_EMULATION_MODE=sw_emu ./nemolite2d.exe
```

Comment:
I haven't got a suitable setup to check this. Can you confirm that you've checked at least the emulation target here?

Reply:
Weird. I'd have thought it was compiler related rather than OS. Anyway, that's not your problem.